cmd/infra/aws/route53.go

```diff
@@ -204,3 +220,22 @@ func cleanRecordName(name string) string {
 	}
 	return s
 }
+
+func retryRoute53WithBackoff(ctx context.Context, fn func() error) error {
+	backoff := wait.Backoff{
+		Steps:    10,
+		Duration: 3 * time.Second,
+		Factor:   1.0,
+		Jitter:   0.1,
```

Comment:
just to be clear, this is "retry every ~3s for up to 30s"

Reply:
the 1.0 factor is an oversight — thoughts on a reasonable set of parameters?

Reply:
If we do `Duration: 1s` and `Factor: 2`, that would be 5 retries at 1s, 2s, 4s, 8s, 16s over 31s

Reply:
I already went with a 1s interval and 1.5 factor, not quite as aggressive in the ramp up but maybe it's good enough to try